x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -212,15 +217,24 @@ export const getColumns = ({
     {
       field: 'tags',
       name: i18n.COLUMN_TAGS,
-      render: (value: Rule['tags']) => (
-        <TruncatableText data-test-subj="tags">
-          {value.map((tag, i) => (
-            <EuiBadge color="hollow" key={`${tag}-${i}`}>
-              {tag}
-            </EuiBadge>
-          ))}
-        </TruncatableText>
-      ),
+      render: (value: Rule['tags']) => {
+        if (value != null && value.length > 0) {
```

Comment:
You shouldn't need to check for `null` here as  `tags` is typed to always return an empty array -- I'm not seeing any errors in my IDE with the following:
<p align="center">
  <img width="400" src="https://user-images.githubusercontent.com/2946766/93539280-889b6d80-f90d-11ea-8b6a-8f0ddae5e697.png" />
</p>